build/BUILD.dawn

```diff
@@ -920,7 +926,7 @@ cc_library(
         # Skia has its own vulkan headers and we do not want these to
         # interfere/override those.
         "-Iexternal/vulkan_headers/include",
-    ],
+    ] + COPTS,
     defines = [
```

Comment:
Use local_defines here (regular defines is still needed for dawn_native_macos since objc_library doesn't have local_defines)